ftw/upgrade/executioner.py

```diff
@@ -46,6 +49,16 @@ def install_upgrades_by_api_ids(self, *upgrade_api_ids):
         data = [(upgrade['profile'], [upgrade['id']]) for upgrade in upgrades]
         return self.install(data)
 
+    security.declarePrivate('_register_after_commit_hook')
+    def _register_after_commit_hook(self):
+
+        def notification_hook(success, *args, **kwargs):
+            result = success and 'committed' or 'aborted'
```

Comment:
@deiferni no :smiling_imp: - that was a conscious choice. `ftw.upgrade` also needs to support Python 2.4. Ternary expressions however were only [introduced in Python 2.5](https://docs.python.org/3.3/faq/programming.html#is-there-an-equivalent-of-c-s-ternary-operator).